pkg/injector/sidecar/container.go

```diff
@@ -202,6 +215,8 @@ func GetSidecarContainer(cfg ContainerConfig) (*corev1.Container, error) {
 			AllowPrivilegeEscalation: ptr.Of(false),
 			RunAsNonRoot:             ptr.Of(cfg.RunAsNonRoot),
 			ReadOnlyRootFilesystem:   ptr.Of(cfg.ReadOnlyRootFilesystem),
+			Capabilities:             &corev1.Capabilities{Drop: []corev1.Capability{"ALL"}},
```

Comment:
This should be set only if the annotation wants this, right? Same for the one below?